apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -161,7 +204,51 @@ class VisualizerModal extends React.Component {
               selectedColumn1={this.state.selectedColumn1}
               selectedColumn2={this.state.selectedColumn2}
             />
+          ) : (
+            <div style={styles.placeholderContainer}>
+              <div style={styles.placeholderText}>
+                Select values to generate a visualization
```

Comment:
i18n

Reply:
Nothing for datasets has been i18n-ized yet. Planning on doing that once things are a little more finalized, since a lot of these strings have changed a couple times. Definitely will be done before this comes out of experiment.